roles/download/defaults/main.yml

```diff
@@ -463,6 +486,15 @@ downloads:
     groups:
       - kube-node
 
+  busybox:
+    enabled: "{{ kube_network_plugin in ['kube-router'] }}"
+    container: true
+    repo: "{{ busybox_image_repo }}"
+    tag: "{{ busybox_image_tag }}"
+    sha256: "{{ busybox_digest_checksum|default(None) }}"
+    groups:
+      - kube-node
```

Comment:
This is used on masters too, as it's the init container of kube-router. So it should prolly be `k8s-cluster` like the kube-router image

Reply:
Thanks, fixed.